src/extractPdfText.ts

```diff
@@ -1,51 +1,59 @@
 import { getPdfPagesCount } from "./getPdfPagesCount";
 import { capitalizeFirstLetter } from "./helperFunctions";
-const pdfLib = require('pdf-to-text');
-// figure out how to deal with this import vs require thing
-// tried modifying tsconfig.json target and module to es2022 and moduleResolution uncomment
-// ^ but didn't work, kept saying module couldn't be found, etc.
+import { join } from "path";
+//@ts-expect-error no type declarations for this module
+import pdfLib from "pdf-to-text";
+import { PdfLib } from "./types";
 
 // Get the user's desired pages range to translate, with a few validity checks
-const getPagesRange = async(pageFrom?: number, pageTo?: number) => {
-    const pagesCount = await getPdfPagesCount();
+const getPagesRange = async (pageFrom = 1, pageTo?: number) => {
+   const pagesCount = await getPdfPagesCount();
 
-    const inputFrom: number = pageFrom !== undefined ? pageFrom : 1;
-    const from: number = inputFrom !< 1 ? inputFrom : 1;
+   const from: number = pageFrom < 1 ? 1 : pageFrom;
 
-    const inputTo: number = pageTo !== undefined ? pageTo : pagesCount;
-    const to: number = inputTo !< pagesCount ? inputTo: pagesCount;
+   const inputTo = pageTo ?? pagesCount;
```

Comment:
the `??` operator defaults to the left variable unless it is `undefined` or `null` in which case it uses the `right`

It is the same as `pageTo !== undefined && pageTo !== null ? pageTo : pagesCount`